packages/ts/form/src/Binder.ts

```diff
@@ -275,7 +275,7 @@ export class Binder<T, M extends AbstractModel<T>> extends BinderNode<T, M> {
 
   protected override update(oldValue: T) {
     if (this[_onChange]) {
-      this[_onChange].call(this.context, oldValue);
+      this[_onChange]!.call(this.context, oldValue);
```

Comment:
Instead of `if` clause you could probably use `?` here (just suggesting).
```ts
protected override update(oldValue: T) {
  this[_onChange]?.call(this.context, oldValue);
}
```